src/main/resources/validationrules/iac/iac-case-role-validation.drl

```diff
@@ -41,6 +42,8 @@ when
              attributes["jurisdiction"] != null && attributes["jurisdiction"].asText() == "IA")
 
 then
+    $ra.setAttribute("jurisdiction", "IA");
```

Comment:
you can write like this $ra.getAttribute().put("jurisdiction", "IA") by doing this no longer need to define setAttribute() in Assignment.

Reply:
Don't think it would work because you need to convert the value as jsonnode.

Reply:
you can call the  method  to convert it in JsonNode from here and it will eliminate the setAttribute() from Assignment Class.